Gemfile

```diff
@@ -1,12 +1,15 @@
 # frozen_string_literal: true
 source "https://rubygems.org"
 
-ruby "2.7.5"
+ruby "3.1.2"
+
+gem "jekyll"
```

Comment:
This shouldn't be needed? GitHub pages brings it in.

Reply:
pushed a change to remove it. looks like i forgot to run an update.